tests/unit/services/onboarding-test.js

```diff
@@ -103,4 +103,47 @@ module('Unit | Service | onboarding', function (hooks) {
 
     assert.verifySteps(['store.createRecord called']);
   });
+
+  test('discordInvite method', async function (assert) {
+    let service = this.owner.lookup('service:onboarding');
+    let fetch = window.fetch;
+    let toast = this.owner.lookup('service:toast');
+
+    toast.error = (message) => assert.step(`toast.error: ${message}`);
+
+    window.fetch = (url, configs) => {
+      assert.step(`fetch called with url: ${url}`);
+
+      if (configs.method === 'POST') {
```

Comment:
Can we be more specific by using API URL in this? Since many APIs can have POST method